ext/lib/crypto/mbedtls/Kconfig.tls-generic

```diff
@@ -160,8 +235,30 @@ config TLS_MAC_SHA256_ENABLED
 config TLS_MAC_SHA512_ENABLED
 	bool "Enable the SHA-384 and SHA-512 hash algorithms"
 
+config TLS_MAC_POLY1305_ENABLED
+	bool "Enable the Poly1305 MAC algorithm"
+
 endmenu
 
+comment "Random number generators"
+
+config TLS_CTR_DRBG_ENABLED
+	bool "Enable the CTR_DRBG AES-256-based random generator"
+	depends on TLS_CIPHER_AES_ENABLED
+	default y
+
+config TLS_HMAC_DRBG_ENABLED
+	bool "Enable the HMAC_DRBG random generator"
+
+comment "Other configurations"
+
+config TLS_CHACHAPOLY_AEAD_ENABLED
```

Comment:
Well, and this is kinda misplaced under "Other configurations", it should go together with similar AEAD ciphers, e.g. TLS_CIPHER_CCM_ENABLED.

Reply:
Agreed, will move